front/src/router/index.js

```diff
@@ -1,14 +1,39 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
-import Home from '@views/Home.vue';
+import { Authorize, UnregisteredUser, RegisteredUser, Admin, Moderator, TextManager } from '@views';
 
 Vue.use(VueRouter);
 
 const routes = [
   {
     path: '/',
-    name: 'Home',
-    component: Home
+    name: 'UnregisteredUser',
```

Comment:
guest

Reply:
And one more thing: is it functionality only for guests? Actually it is common for all users. So, it is common functionality for all users, so it is not correct to say that it is guest or unregistered (why un**registered**, what about registered but unauthorized users)